announcements/1.12.0.md

```diff
@@ -0,0 +1,64 @@
+## Introduction
+
+The Anbox Cloud team is pleased to announce the release of Anbox Cloud 1.12.0.
+
+Please see [component versions](https://anbox-cloud.io/docs/component-versions) for a list of updated components.
+
+## New Features & Improvements
+
+### Android 12
+
+Google [has released the final version Android 12 on October 4](https://android-developers.googleblog.com/2021/10/android-12-is-live-in-aosp.html) to the Android Open Source Project and its now shipping as part of Anbox Cloud 1.12
+
+Anbox Cloud supports the same features across all supported Android versions, so everything you have used so far on other Android versions in Anbox Cloud is available on Android 12 too.
+
+### Improved density for NVIDIA GPUs
+
+On supported NVIDIA GPUs (Tesla T4) Anbox Cloud can now deliver higher density. On the NVIDIA Tesla T4, Anbox Cloud can now provide up to 30 containers all rendering and streaming at 720p 30 FPS.
+
+### WiFi support
+
+WiFi support got improved and is now more stable. With 1.12 WiFi is still disabled by default but can be enabled with the `enable_wifi` feature flag. In 1.13 WiFi will be enabled by default for all containers.
+
+### Rebooting Android is now possible
+
+So far reboots inside Android running in the Anbox container was strictly not allowed. With 1.12 it is now possible to allow Android to reboot itself. During the reboot the Anbox runtime stays alive and continues to be functional (e.g. a connected WebRTC stream stays active).
```

Comment:
```suggestion
So far, rebooting the Android running in the Anbox container was not allowed. With 1.12, it is now possible to allow Android to reboot. During the reboot, the Anbox runtime stays alive and continues to be functional (for example, a connected WebRTC stream stays active).
```

Reply:
I'm a bit unsure about the "strictly". If you want a stronger statement than just "not allowed", you could use "... was strictly forbidden". But "strictly not allowed" sounds a bit strange. ;)